dashboards-reports/public/components/main/report_definition_details/report_definition_details.tsx

```diff
@@ -426,12 +426,12 @@ export function ReportDefinitionDetails(props: { match?: any; setBreadcrumbs?: a
       reportHeader:
         reportParams.core_params.hasOwnProperty('header') &&
         reportParams.core_params.header != ''
-          ? converter.makeMarkdown(reportParams.core_params.header)
```

Comment:
Let's remove the import if not used anymore.

Reply:
same for `report_details.tsx`

Reply:
Removed import from both files